src/screens/Login.js

```diff
@@ -118,39 +134,65 @@ class Login extends Component {
-        <View style={styles.backgroundStyle}>
-          <View style={styles.viewStyle}>
-            <OpacityButton text="Play without logging!" onPress={this.onAnonymous} />
-            <Text style={styles.signUpText}>
-              Sign up/in
-            </Text>
-            <Text style={styles.infoText}>
-              After logging you would see history of your plays
-            </Text>
-            <StatusBar barStyle="light-content" />
+      <ImageBackground
+        style={styles.backgroundImage}
+        source={require('./../../assets/background.png')}
+      >
+        <KeyboardAvoidingView behavior="padding" style={[styles.view, styles.keyboardAvoidingView]}>
+          <View style={[styles.view, styles.logoView]}>
+            <Image
+              resizeMode="contain"
+              style={styles.logo}
+              source={require('./../../assets/logo.png')}
+            />
+          </View>
+          <View style={[styles.view, styles.inputView]}>
             <TextInput
-              style={styles.inputStyle}
+              style={styles.input}
+              placeholderTextColor="white"
               keyboardType="email-address"
               onChangeText={this.onEmailChange}
-              placeholder="E-mail"
+              placeholder="Email"
             />
+          </View>
+          <View style={[styles.view, styles.inputView]}>
             <TextInput
-              style={styles.inputStyle}
+              style={styles.input}
+              placeholderTextColor="white"
               onChangeText={this.onPasswordChange}
               secureTextEntry
               placeholder="Password"
             />
-            <View style={styles.controlPanel}>
-              <View style={styles.buttonsLayout}>
-                <OpacityButton text="Sign in" onPress={this.onLogin} />
-                <OpacityButton text="Sign up" onPress={this.onRegister} />
-              </View>
-              {!!error && <Error error={error} />}
-            </View>
           </View>
-        </View>
-      </KeyboardAvoidingView>
+          {!!error && <Error error={error} />}
+          <View style={[styles.loginView]}>
+            <LinearGradient colors={[localColors.redGradient1, localColors.redGradient2]}>
+              <TouchableOpacity
+                style={[styles.loginButton]}
+                onPress={this.onLogin}
+              >
+                <Text style={buttonStyles.buttonText}>
+                  LOGIN
+                </Text>
+              </TouchableOpacity>
+            </LinearGradient>
+          </View>
+          <View style={[styles.view, styles.signUpInfoView]}>
+            <Text style={[styles.signUpInfoText, styles.signUpInfoText1]}>
+              No   account   yet?
+            </Text>
+            <Text
+              style={[styles.signUpInfoText, styles.signUpInfoText2]}
+              onPress={() => {
+                const { navigation: { navigate } } = this.props;
```

Comment:
This getting navigation object could be encapsulated maybe? Let's maybe put this to a seperate function like `getNavigator`?